### PR TITLE
misc cleanup

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* None

*Description of changes:*
- removed unneeded icon font preload
- set section icons to blue
- updated isSame() same as other projects
- delete obsolete common mixin methods
- updated package versions to match lock file
- updated checkmarks to the designed colour

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,57 +12,57 @@
     "test:debug": "node --inspect-brk node_modules/.bin/vue-cli-service test:unit --testPathPattern --no-cache --watch --runInBand"
   },
   "dependencies": {
-    "@babel/compat-data": "^7.8.6",
-    "@mdi/font": "^5.0.45",
-    "@sentry/browser": "^5.15.4",
-    "@sentry/integrations": "^5.15.4",
+    "@babel/compat-data": "^7.11.0",
```
Comment (severinbeauvais) [inline package.json:15]: These are the versions actually used (per the lock file).
Updating these has no impact.

File: src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue [hunk 2]
```diff
@@ -12,7 +12,7 @@
     <div id="people-roles-summary" v-if="isSummary">
       <!-- Summary Header -->
       <div class="people-roles-summary-header" >
-        <v-icon>mdi-account-multiple-plus</v-icon>
+        <v-icon color="#38598A">mdi-account-multiple-plus</v-icon>
```
Comment (severinbeauvais) [inline src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue:15]: Same as Edit UI.

**Review by lekshmimallika-aot — APPROVED** [review]
Nice!

File: src/views/CreateShareStructure.vue [hunk 3]
```diff
@@ -45,8 +45,8 @@
     <p>Your application must include the following:</p>
     <ul>
       <li>
-        <v-icon color="blue" v-if="shareClasses.length > 0" class="cp-valid">mdi-check</v-icon>
-        <span v-else><v-icon color="red" v-if="showErrors" class="cp-invalid">mdi-close</v-icon></span>
+        <v-icon v-if="shareClasses.length > 0" color="green darken-2" class="cp-valid">mdi-check</v-icon>
+        <v-icon v-else :color="showErrors ? 'red': 'transparent'" class="cp-invalid">mdi-close</v-icon>
```
Comment (severinbeauvais) [inline src/views/CreateShareStructure.vue:49]: This is the same fix I did for People and Roles so the icon takes up space but doesn't display anything.

File: src/components/IncorporationAgreement/AgreementType.vue [hunk 4]
```diff
@@ -4,27 +4,27 @@
       <v-card flat>
         <!-- Summary Header -->
         <div class="agreement-summary-header">
-            <v-icon>mdi-handshake</v-icon>
-            <label class="agreement-summary-title">
-                <strong>Incorporation Agreement and Benefit Company Articles</strong>
-            </label>
+          <v-icon color="#38598A">mdi-handshake</v-icon>
```
Comment (cameron-freshworks) [inline src/components/IncorporationAgreement/AgreementType.vue:7]: If we were to set the icon colour in the base style for v-icons would it inherit throughout? 
I know Vuetify was tricky sometimes about inheritance, if color was a prop for instance.

**Review by cameron-freshworks — APPROVED** [review]
Looks good, just one question. Merge away when ready 👍

File: src/components/IncorporationAgreement/AgreementType.vue (hunk 4 above)
Comment (severinbeauvais) [reply src/components/IncorporationAgreement/AgreementType.vue:7]: That's clever.

There are many interesting things on the following page but I don't see any about the icons :/  Also we've been using some v-icons with conditional colours, which we would have to keep doing.

https://vuetifyjs.com/en/features/theme/#customizing

However, I am aware of some "style consistency" tickets in the backlog. When we work on them, we should look into this a bit deeper. If the Vuetify theme won't help us directly, I think we should consider a common set of classes, eg "v-icon-check-ok", "v-icon-header", "v-btn-remove", etc.

I can only find a couple of these "style" tickets atm (#5368, #5470) so we'll have to remember to do this ^^ when we work on them.

**Pull request merged**